src/Stryker.ts

```diff
@@ -42,30 +41,27 @@ export default class Stryker {
     this.applyConfigWriters();
     this.setGlobalLogLevel(); // loglevel could be changed
     this.freezeConfig();
-    var reporterFactory = new ReporterFactory();
-    this.reporter = reporterFactory.getReporter('console');
   }
 
   /**
    * Runs mutation testing. This may take a while.
    * @function
    */
-  runMutationTest(): Promise<void> {
+  runMutationTest(): Promise<MutantResult[]> {
```

Comment:
Why are we returning a `MutantResult[]`, just because we can and because the calling process may want to do something with it?